src/main/java/slackIntegration/EyesSlack.java

```diff
@@ -0,0 +1,38 @@
+package slackIntegration;
+
+import com.github.seratch.jslack.Slack;
+import com.github.seratch.jslack.api.webhook.Payload;
+import com.github.seratch.jslack.api.webhook.WebhookResponse;
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.IOException;
+import java.util.List;
+
+public class EyesSlack {
+
+    private static final String urlSlackWebHook = "https://hooks.slack.com/services/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";
+    private static final String channelName = "slackIntegration";
+
+    public static void sendTestExecutionStatusToSlack(List<String> passedTests, List<String> failedTests,
```

Comment:
method brakes SRP principle because is responsible for message creation and sending request. Please split it or extract the creation part